fabric-networking-api-v1/src/main/java/net/fabricmc/fabric/mixin/networking/CustomPayloadS2CPacketMixin.java

```diff
@@ -63,4 +77,16 @@ private static PacketCodec<PacketByteBuf, CustomPayload> wrapConfigCodec(CustomP
 		fabricCodec.fabric_setPacketCodecProvider((packetByteBuf, identifier) -> PayloadTypeRegistryImpl.CONFIGURATION_S2C.get(identifier));
 		return codec;
 	}
+
+	@Override
+	public void fabric_split(int id, NetworkState<?> state, ChannelHandlerContext channelHandlerContext, EncoderHandler<?> encoder, Packet<?> packet, Consumer<Packet<?>> consumer) throws Exception {
+		int size = (state.id() == NetworkPhase.CONFIGURATION ? PayloadTypeRegistryImpl.CONFIGURATION_S2C : PayloadTypeRegistryImpl.PLAY_S2C).getSplittingThreshold(this.payload.getId());
```

Comment:
Nit: split this into 2 lines, create another variable for the payload type registry. easier to read.